INSTALL.md

```diff
@@ -0,0 +1,100 @@
+## Installation instructions
+
+In order to run the site in developement mode you need some packages to be installed.
+
+#### Installing `sphinx` into a `virtualenv`
+
+At first, you need to have `sphinx` properly working on your environment. It is needed
+because the next step is to build the documentations locally. 
+
+We suggest to install it into a `virtualenv` and for doing this we first need an environment.
+Let's check if you have correctly installed `virtualenv` by running:
+
+```bash
+which virtualenv
+```
+
+If yes, you will read its path location on the terminal. If not, you can install it following:
+
+```bash
+pip install virtualenv
+```
```

Comment:
Also for this one, `virtualenv` has its own installation page https://virtualenv.pypa.io/en/latest/installation.html, so don't repeat the work, and just redirect people there (in this case is trivial, but you never know when things change, so better to properly delegate)